addons/gloot/ctrl_inventory_grid.gd

```diff
@@ -49,7 +49,7 @@ func _set_inventory(new_inventory: InventoryGrid) -> void:
     _connect_signals()
 
 
-func _ready():
+func _enter_tree():
```

Comment:
As stated in the previous comment, this doesn't seem to fix the underlying issue - `_set_inventory()` should not be called before `_ctrl_item_container` is set.
Moving line 58 below line 67 should fix this.